src/components/CardPicker.js

```diff
@@ -43,12 +43,18 @@ type CardPickerProps = {
    * the width of each card. Defaults to 3 columns.
    */
   columns: number;
+  /** 
+   * Property that contains the prefix for data-test attribute added to elements to be uniquely
+   * identified by testing tools like Selenium
+   */
+  dataTestPrefix : string | null;
```

Comment:
This should be optional. Reminder that `null` and `undefined` are not equivalent. This typing indicates a user must always specify `dataTestPrefix`, but the value may be a string or else explicitly defined as `null`.

Instead, this should be: `dataTestPrefix?: string;
Which indicates the prop is optional, but must be a string if specified.